### PR TITLE
#1246 - Doc for keyword *bonds* of PDBWriter is wrong - attempt at fix

### LINKED ISSUE #1246: Doc for keyword *bonds* of PDBWriter is wrong
The documentation of `PDBWriter.__init__` tells:

```
*bonds*
           write bonds to the PDB file as CONECT_ records [``False``]
```

This is in contradiction with the signature of the signature that tells that the default value for the argument is `"conect"`. It is also in contradiction with the code of `PDBWriter._write_pdb_bonds`:

```python
        # Write out only the bonds that were defined in CONECT records
        if self.bonds == "conect":
            bonds = ((bond[0].index, bond[1].index) for bond in bondset if not bond.is_guessed)
        elif self.bonds == "all":
            bonds = ((bond[0].index, bond[1].index) for bond in bondset)
        else:
            raise ValueError("bonds has to be either None, 'conect' or 'all'")
```

### PR DESCRIPTION
**Pull request by Shtkddud123**
Fixes #1246 

Changes made in this Pull Request:
 -  The __init__ documentation for bonds has been fixed in a numpy style. 
PR Checklist
------------
 - [ X] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

File: package/MDAnalysis/coordinates/PDB.py [hunk 1]
```diff
@@ -490,44 +490,42 @@ class PDBWriter(base.WriterBase):
     def __init__(self, filename, bonds="conect", n_atoms=None, start=0, step=1,
                  remarks="Created by PDBWriter",
                  convert_units=None, multiframe=None):
+
         """Create a new PDBWriter
+        
+         Note
+         ____
```
Comment (jbarnoud) [inline package/MDAnalysis/coordinates/PDB.py]: Should be dashes rather than underscores.

File: package/MDAnalysis/coordinates/PDB.py [hunk 2]
```diff
@@ -490,44 +490,42 @@ class PDBWriter(base.WriterBase):
...
-              if bond information is available in the topology. (For selections
-              smaller than the whole :class:`Universe`, the atom numbering in
-              the CONECT_ records would not match the numbering of the atoms in
-              the new PDB file and therefore a :exc:`NotImplementedError` is
-              raised.)
-
-         *multiframe*
+           the value of :data:`MDAnalysis.core.flags` ['convert_lengths']      
+         bonds: :obj:, 'str', optional
+           If set to "conect", then write those bonds to the output file that were already defined in an input PDB file as PDB CONECT_ record.           If set to "all", write all bonds (including guessed ones) to the file. ``None`` does not write any bonds. The default is "conect".
```
Comment (jbarnoud) [inline package/MDAnalysis/coordinates/PDB.py]: There is a large hole between "records." and "If".

File: package/MDAnalysis/coordinates/PDB.py [hunk 3]
```diff
@@ -490,44 +490,42 @@ class PDBWriter(base.WriterBase):
...
-         *multiframe*
+           the value of :data:`MDAnalysis.core.flags` ['convert_lengths']      
+         bonds: :obj:, 'str', optional
+           If set to "conect", then write those bonds to the output file that were already defined in an input PDB file as PDB CONECT_ record.           If set to "all", write all bonds (including guessed ones) to the file. ``None`` does not write any bonds. The default is "conect". 
+         multiframe: int
            ``False``: write a single frame to the file; ``True``: create a
            multi frame PDB file in which frames are written as MODEL_ ... ENDMDL_
            records. If ``None``, then the class default is chosen.    [``None``]
 
-        .. _CONECT: http://www.wwpdb.org/documentation/format32/sect10.html#CONECT
```
Comment (jbarnoud) [inline package/MDAnalysis/coordinates/PDB.py]: These lines link the occurrences of "_CONECT", "_MODEL", and "_ENDMDL" to the appropriate URL. They should not be removed.

File: package/MDAnalysis/coordinates/PDB.py [hunk 4]
```diff
@@ -1000,6 +998,23 @@ def CONECT(self, conect):
         self.pdbfile.write(self.fmt['CONECT'].format(conect))
 
 
+class PrimitivePDBReader(PDBReader):
```
Comment (jbarnoud) [inline package/MDAnalysis/coordinates/PDB.py]: This class and the next one were removed in a previous commit. They should not be reintroduced. Please [rebase your branch](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) against the develop branch so it is up to date.

File: package/MDAnalysis/lib/distances.py [hunk 5]
```diff
@@ -206,7 +206,7 @@ def _check_lengths_match(*arrays):
     if not all( a.shape == ref for a in arrays):
         raise ValueError("Input arrays must all be same shape"
                          "Got {0}".format([a.shape for a in arrays]))
-
+
```
Comment (jbarnoud) [inline package/MDAnalysis/lib/distances.py]: Please remove the trailing spaces.

**Comment by orbeckst** [comment]
I fixed up the PR and resubmitted as PR #1329

**Pull request closed without merging**